eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentEvents.scala

```diff
@@ -144,7 +144,7 @@ object PaymentFailure {
    */
   def transformForUser(failures: Seq[PaymentFailure]): Seq[PaymentFailure] = {
     failures.map {
-      case LocalFailure(hops, AddHtlcFailed(_, _, t, _, _, _)) => LocalFailure(hops, t) // we're interested in the error which caused the add-htlc to fail
+      case LocalFailure(hops, t) => LocalFailure(hops, t) // we're interested in the error which caused the add-htlc to fail
```

Comment:
Now that this is just a passthrough, the whole `.map` is unnecessary, you can remove it:
```
failures match {
      case previousFailures :+ LocalFailure(_, RouteNotFound) if previousFailures.nonEmpty => previousFailures
      case other => other
}
```

Reply:
Hawk eyes 👍 . Done with c46fff704.